libwnbd/wnbd_ioctl.cpp

```diff
@@ -133,6 +137,245 @@ DWORD WnbdOpenAdapterCMDeviceInstance(PDEVINST DeviceInstance)
     return Status;
 }
 
+DWORD GetDrvDetailData(HDEVINFO DevHandle, PSP_DEVINFO_DATA DevInfoData,
```

Comment:
nit: let's rename DevHandle to AdapterHandle or AdapterDevHandle, underlining the fact that this isn't actually a disk device.